pkg/storage/storage.go

```diff
@@ -16,6 +16,17 @@ import (
 	"github.com/pkg/errors"
 )
 
+// GetStorage returns Storage defination for given Storage name
+func GetStorage(storageName string, storages StorageList) Storage {
```

Comment:
This looks weird. 
It might be better to have `StorageList` as a receiver. What do you think?  `func (sl *StorageList) Get(name string) Storage`

Reply:
looks good to me